### ISSUE TITLE
Another flaw

### ISSUE THREAD

**Issue by poetslipvis** [rualark/AiHarmony#8]
When you click the non HD play button, that is, the midi button (I think), you can't stop it (or at least I don't know how). A stop and pause button would be nice. 

Another problem is that, during the playback, when you click on the play button again, another playback begins, so now you have two songs playing at different moments, and if you click again, a third one, and so forth.

**Comment by rualark** [comment]
Non-HQ playback is still in development. So far it works in this "beta" way. I plan to improve it (it is on my kaban board).

**Comment by rualark** [comment]
Implemented
